caffeinated/modules/ticket_selector_caff/templates/ticket_selector_price_details.template.php

```diff
@@ -14,122 +34,112 @@
-                            <?php $running_total += $tax_amount; ?>
+                            <?php $pretax_total += ! $price_mod->is_tax() ? $new_sub_total : 0; ?>
+                            <?php $running_total += $new_sub_total; ?>
                         </tr>
                     <?php } ?>
-                <?php } ?>
-                <tr>
-                    <td colspan="2" class="jst-rght small-text ttl-lbl-td">
-                        <b><?php
-                            echo apply_filters(
-                                'FHEE__ticket_selector_chart_template__ticket_details_total_price',
-                                esc_html__('Total', 'event_espresso')
-                            ); ?></b></td>
-                    <td data-th="<?php
-                                    echo apply_filters(
-                                        'FHEE__ticket_selector_chart_template__ticket_details_total_price',
-                                        esc_html__('Total', 'event_espresso')
-                                    ); ?>" class="jst-rght small-text">
-                        <b><?php echo EEH_Template::format_currency($running_total); ?></b></td>
-                </tr>
+                    <?php if ($ticket->taxable()) {
+                        echo espressoSubtotalRow($running_total);
+                        foreach ($ticket->get_ticket_taxes_for_admin() as $tax) { ?>
+                            <tr>
+                                <td data-th="<?php esc_html_e('Name', 'event_espresso'); ?>"
+                                    class="jst-rght small-text"
+                                ><?php echo $tax->name(); ?></td>
+                                <td data-th="<?php esc_html_e('Description', 'event_espresso'); ?>"
+                                    class="jst-rght small-text"
+                                ><?php echo $tax->amount(); ?>%
+                                </td>
+                                <?php $tax_amount = $pretax_total * ($tax->amount() / 100); ?>
+                                <td data-th="<?php esc_html_e('Amount', 'event_espresso'); ?>"
+                                    class="jst-rght small-text"
+                                >
+                                    <?php echo EEH_Template::format_currency($tax_amount); ?>
+                                </td>
+                                <?php $running_total += $tax_amount; ?>
+                            </tr>
+                        <?php } ?>
+                    <?php } ?>
+                    <tr>
+                        <td colspan="2" class="jst-rght small-text ttl-lbl-td">
+                            <b>
+                                <?php echo apply_filters(
+                                    'FHEE__ticket_selector_chart_template__ticket_details_total_price',
+                                    esc_html__('Total', 'event_espresso')
+                                ); ?>
```

Comment:
Should `esc_html__` be run before applying the filters or afterwards?

Reply:
good call... will fix, thnx